src/app/app.component.ts

```diff
@@ -1,10 +1,81 @@
-import { Component } from '@angular/core';
-
+import { Component, OnInit  } from '@angular/core';
+import { discardPeriodicTasks } from '@angular/core/testing';
+import { Injectable, Pipe, PipeTransform } from '@angular/core';
+import{SearchFilterPipe} from './filter.pipe'
 @Component({
   selector: 'app-root',
+
   templateUrl: './app.component.html',
-  styleUrls: ['./app.component.css']
+  styleUrls: ['./app.component.css'],
+
 })
-export class AppComponent {
-  title = 'app';
+export class AppComponent implements OnInit{
+  tasks: Task[] = [];
+  newTask = "  ";
+  id = 1;
+  itemFilter: any = { text: '' };
+  tasksDone = [];
+   title:String;
+constructor() { 
+  this.title = " ";
+  this.tasks = []
+}
+
+ngOnInit() {
+}
+
+
+
+  pushTask = function () {
+    // jesli ten element jest pusty dodaje nowy task
+    if (this.newTask) {
+      const newTaskObject: Task = {
+        title: this.newTask,
+        date: new Date(),
+        done: false
+      };
+      this.tasks.push(newTaskObject);
+      this.newTask = " ";
+    }
+  }
+
+  // usuwanie
+  removeTask = function (index) {
+    this.tasks.splice(index, 1);
+  }
+  removeTask1 = function (index) {
+    this.tasksDone.splice(index, 1);
+
+  }
+  done(task: string, ind) {
```

Comment:
Po jednej pustej linijce pomiędzy metodami. Dlaczego skoro `task` ma typ to `ind` go nie ma? (swoją drogą też słaba nazwa)

Reply:
https://github.com/jedi800/Checklistaa/pull/1/commits/d7a95d42eab1ce1bddbaa6add2f2f16adbe3750d